### ISSUE TITLE
rename tokens

### ISSUE THREAD

**Issue by xeno-by** [scalameta/scalameta#350]
I've always been kinda negative towards having to type countless backticks to write anything involved that deals with tokens. This is something that I'd like to finally fix before releasing 0.1.

**Comment by olafurpg** [comment]
My 2 cents.

I don't agree that it's annoying to write the back ticks. And even if it is annoying, I think you should try to optimize for readable code instead of write friendly code. My experience with the backtick token classes is that they are very readable.

However, I don't mind if you rename the tokens to use alphabetical names. Also, the backtick names can look scary for newcomers and they do have some problems:
```
`_ ` is weird because of the extra space
`{` `(` `[` and their closing counterparts mess up with finding "matching parenthesis"
in text editors that use basic tokenizers
```

**Comment by densh** [comment]
I completely disagree with this. Backticks are crucial for readability of token-processing code. They are super clear and easy to read. This is confirmed by scalafmt codebase.

**Comment by xeno-by** [comment]
Merged into #372.
